src/bidiMapper/domains/context/PreloadScriptStorage.spec.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {expect} from 'chai';
+import sinon from 'sinon';
+
+import {PreloadScriptStorage} from './PreloadScriptStorage';
+
+describe('PreloadScriptStorage', () => {
+  let preloadScriptStorage: PreloadScriptStorage;
+
+  beforeEach(() => {
+    preloadScriptStorage = new PreloadScriptStorage();
+  });
+
+  it('initial state', () => {
```

Comment:
I assume you wanted to add some more tests

Reply:
Only later on, once this class is stable. This is purposely just skeleton code for now.

Reply:
I included a TODO at the top.